docs/sections/fundamentals/basics.md

```diff
@@ -249,65 +257,144 @@ As a final step, let's have a look at the provenance of this simple calculation.
 
-(fig-calcfun-graph)=
+To get more information about the inputs, outputs, etc. of this calculation, we can just do as indicated again and pass the specific entry point as an additional argument for the command:
 
-```{figure} include/images/basics_calcfun_graph.png
-:scale: 50
-:align: center
+```{code-block} console
 
-Provenance graph of the `multiply` calculation function.
+$ verdi plugin list aiida.calculations arithmetic.add
+Description:
+
+	`CalcJob` implementation to add two numbers using bash for testing and demonstration purposes.
+
+Inputs:
+           code:  required  Code             The `Code` to use for this job.
+              x:  required  Int, Float       The left operand.
+              y:  required  Int, Float       The right operand.
+       metadata:  optional
+Outputs:
+  remote_folder:  required  RemoteData       Input files necessary to run the process will be stored in this folder node ...
+      retrieved:  required  FolderData       Files that are retrieved by the daemon will be stored in this node. By defa ...
+            sum:  required  Int, Float       The sum of the left and right operand.
+   remote_stash:  optional  RemoteStashData  Contents of the `stash.source_list` option are stored in this remote folder ...
+Exit codes:
+              1:  The process has failed with an unspecified error.
+              2:  The process failed with legacy failure mode.
+             11:  The process did not register a required output.
+            100:  The process did not have the required `retrieved` output.
+            110:  The job ran out of memory.
+            120:  The job ran out of walltime.
+            310:  The output file could not be read.
+            320:  The output file contains invalid output.
+            410:  The sum of the operands is a negative number.
 
 ```
 
-## CalcJobs
+There are several important things to note from the obtained output.
+The first is description of the calculation, which explains that it adds up two numbers together.
+Then there are the inputs, of which 3 are required: the code to be used to add the numbers up, and the two numbers to be added up.
+Finally, among the outputs, you can see the `sum` which contains the result of the addition.
 
-When running calculations that require an external code or run on a remote machine, a simple calculation function is no longer sufficient.
-For this purpose, AiiDA provides the `CalcJob` process class.
-To run a `CalcJob`, you need to set up two things: a `code` that is going to implement the desired calculation and a `computer` for the calculation to run on.
+Now that we understand what our `CalcJob` does and what it needs, let us see what we need to do to actually execute it.
+
+### Preliminary setups
+
+Before you run a `CalcJob`, you need to have two things: a `code` that is going to implement the desired calculation and a `computer` for the calculation to run on.
```

Comment:
The `localhost` computer will already be set up, and we don't want to use the `tutor` computer anymore. It just made things more confusing in the last tutorial. :)

Reply:
I was testing this in the Quantum Mobile and tried setting up the code for `localhost` but my runs excepted. I suspected problems with SLURM but I'm not sure.

Also, since I can't connect to the AiiDAlab I couldn't check if it comes with any computer pre-configured or not.

Reply:
Sure, no worries. But on the JupyterHub cluster the `localhost` will be set up, so no need to add this here. :)